lib/api_bridge/query.rb

```diff
@@ -24,6 +24,7 @@ def initialize url, facets=[], options={}
         # TODO how to incorporate facets here and facets from request
         # into query?  See also: f[] field
         @facet_fields = facets
+        options = _remove_rows(options)
         @options = override_options options
```

Comment:
This is tangential to the PR content, but figured I'd mention here anyway.

I think we'd want this written as `override_options(options)` since it's our own method and not one of the Rails DSLs.

Also this line is assigning to `@options`, but the last line in the `override_options` method is also assigning to `@options` without a separate final line returning the object. I'm curious/concerned that the return value from that last assignment in `override_options` which is then in turn assigned again to `@options` here may get mixed up, or at least possibly be confusing to read later. Maybe `override_options` should just return `ApiBridge.override_options(existing, requested)` and not assign to `@options` on its last line?

Reply:
Ah, well I didn't want to make those types of changes to api_bridge, to be honest, but I'm really tearing into the version that I'm putting in orchid right now.  I've added parentheses.  I have actually already torn out the query override_options method because I didn't like how it was put together particularly.  I think the orchid version is going to be much better  :)

Perhaps I should have made all those changes to the api_bridge gem itself, but since it's going to be considered deprecated I decided okay, just make a minimal change to make sure that bug was fixed, and otherwise leave it alone.  What do you think?

Reply:
I think that's fine to leave things less re-worked here if they're getting a good overhaul elsewhere. As long as we have something that points us back to the older history of the code here just in case, I think that'll be good.